code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -614,24 +614,33 @@ emp_act
 
 			playsound(loc, 'sound/weapons/slice.ogg', 25, TRUE, -1)
 			visible_message("<span class='danger'>[M] has slashed at [src]!</span>", \
- 				"<span class='userdanger'>[M] has slashed at [src]!</span>")
+				"<span class='userdanger'>[M] has slashed at [src]!</span>")
 
 			apply_damage(damage, BRUTE, affecting, armor_block)
 			add_attack_logs(M, src, "Alien attacked")
 			updatehealth("alien attack")
+			#define ARM_MULTI 1.15
```

Comment:
You're not in *that* much of a hurry, spell this baby out, no ambiguous shortcuts.

Also, I putting this in the middle of a proc's code is a bit messy. Put it above the proc or at the start of the file.